src/library/Uno.Material/Styles/Controls/CheckBox.xaml

```diff
@@ -124,9 +124,9 @@
 								<VisualState x:Name="UncheckedDisabled">
 									<VisualState.Setters>
 										<Setter Target="BackgroundBorder.Opacity"
```

Comment:
we should use the proper specs with the color scheme. I don't think hardcoding 0.5 is acceptable

Reply:
Zeplin specifies nothing for some of the controls, I simply put a generic value to show disabled. Should every control author revisit the Disabled state, it seems as if they were all skipped because my old values were still in place.

Reply:
I'll open a GitHub task to review Disabled states across the controls.